ajax/libs/js-popup/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "js-popup",
+  "filename": "js-popup.min.js",
+  "version": "1.0.0",
+  "description": "Simple popup messages for JavaScript/ReactJS.",
+  "homepage": "https://github.com/exelban/js-popup/",
+  "keywords": [
+    "react",
+    "reactjs",
+    "popup",
+    "popup-window",
+    "popup-messages",
+    "popup-windows",
+    "popup-messaging",
+    "window",
+    "window-popup",
+    "popups",
+    "js-popup",
+    "component",
+    "JavaScript",
+    "box/popup window",
+    "box window",
+    "dialog window",
+    "dialog popup",
+    "dialog popup window"
+  ],
+  "author": {
+    "name": "Serhiy Mytrovtsiy",
+    "email": "mitrovtsiy@ukr.net"
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/exelban/js-popup.git"
+  },
+  "npmName": "js-popup",
+  "npmFileMap": [
+    {
+      "basePath": "release",
+      "files": [
+        "*"
```

Comment:
`files` is a pattern matcher allowing selection of multiple files, see [node-glob](https://github.com/isaacs/node-glob#glob-primer)
it  will be better to modify as the following:
`"*"` --> `"*/**"`